src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -24,6 +25,7 @@
   private static final String DIRECTORS_PARAMETER_KEY = "Director";
   private static final String WRITERS_PARAMETER_KEY = "Writer";
   private static final String ACTORS_PARAMETER_KEY = "Actors";
+  private static final String OMDB_ID_PARAMETER_KEY = "imdbID";
```

Comment:
nit: omdbId just to be consistent? also ID => Id

Reply:
Oh yeah the reason it is "imdbID" with both letters capitalized is because that is the format that the Imdb object has for the property of that name. If I change the capitalization the request won't be able to find that parameter.

Reply:
maybe rename `OMDB_ID_PARAMETER_KEY` to `IMDB_ID_PARAMETER_KEY` since that is what it actually represents?

Reply:
OK I renamed the parameter key to represent the object sent in the response.